compiler/optimizations/gpuTransforms.cpp

```diff
@@ -1311,3 +1378,137 @@ void gpuTransforms() {
     }
   }
 }
+
+bool isLoopGpuBound(CForLoop* loop) {
+  return eligibleLoops.find(loop) != eligibleLoops.end();
+}
```

Comment:
Could this be implemented by just checking the `PRIM_GPU_ELIGIBLE` primitive in the loop body?